visualization/app/codeCharta/ui/colorPickerForMapColor/mapColorLabel.pipe.ts

```diff
@@ -5,14 +5,35 @@ import { MetricMinMax } from "../../state/selectors/accumulatedData/metricData/s
 
 @Pipe({ name: "mapColorLabel" })
 export class MapColorLabelPipe implements PipeTransform {
-	transform(metricName: keyof MapColors, colorRange: ColorRange, nodeMetricRange: MetricMinMax): string {
+	transform(metricName: keyof MapColors, colorRange: ColorRange, nodeMetricRange: MetricMinMax, colorMetric: string): string {
 		switch (metricName) {
-			case "positive":
-				return `${nodeMetricRange.minValue} to < ${this.formatNumber(colorRange.from)}`
-			case "neutral":
-				return `≥ ${this.formatNumber(colorRange.from)} to ≤ ${this.formatNumber(colorRange.to)}`
-			case "negative":
-				return `> ${this.formatNumber(colorRange.to)} to ${this.formatNumber(nodeMetricRange.maxValue)}`
+			case "positive": {
+				const isColorMetricUnary = colorMetric === "unary"
+				const isFromValueEqualMinValue = nodeMetricRange.minValue === colorRange.from
+				const isFromValueEqualMaxValue = nodeMetricRange.maxValue === colorRange.from
+				return isColorMetricUnary
+					? `${nodeMetricRange.minValue} - ${nodeMetricRange.minValue}`
+					: isFromValueEqualMinValue
+					? `-`
+					: isFromValueEqualMaxValue
+					? `${nodeMetricRange.minValue} to ${this.formatNumber(colorRange.from)}`
+					: `${nodeMetricRange.minValue} to ${this.formatNumber(colorRange.from - 1)}`
```

Comment:
It is hard to read this nested if conditions. Can be add parantheses around each condition?